src/main/kotlin/org/jitsi/nlj/Transceiver.kt

```diff
@@ -87,7 +87,7 @@ class Transceiver(
 
     private val transportCcEngine = TransportCcEngine(bandwidthEstimator, logger)
 
-    private val rtpSender: RtpSender = RtpSenderImpl(
+    private val rtpSender: RtpSenderImpl = RtpSenderImpl(
```

Comment:
i've added setFeature methods to the rtpSender/rtpReceiver abstract classes temporarily, but i'll switch to an event based approach because it seems better